### ISSUE TITLE
Subs & Dubs missing

### ISSUE THREAD

**Issue by Akamaru** [OpenSTDL/CrunchyDL#60]
**Describe the bug**
When I try to download new episodes from today, all dubs and subs are missing

**To Reproduce**
1. Open CRD
2. Add Crunchyroll link to download
3. In select screen Dubs and Subs are missing

**Expected behavior**
Download Dubs and Subs

**Screenshots**
![grafik](https://github.com/OpenSTDL/CrunchyDL/assets/8063440/ce95533c-d0e4-44f7-a9be-7b84534944bd)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser None
 - Version CRD v1.2.4

**Additional context**
It happens for "[My Deer Friend Nokotan](https://www.crunchyroll.com/de/series/G0XHWM100/my-deer-friend-nokotan)" and "[Wistoria: Wand and Sword](https://www.crunchyroll.com/de/series/GW4HM7WK9/wistoria-wand-and-sword)"

**Comment by Fu2chN** [comment]
![image](https://github.com/OpenSTDL/CrunchyDL/assets/13276983/891784f2-70b4-49bd-bbd0-21bb2e8b6f05)

No problem for me for this anime but for "Wistoria Wand and Sword" I have this.

![image](https://github.com/OpenSTDL/CrunchyDL/assets/13276983/f47cc4ca-3fc5-42a9-9408-67f2dc56a47e)

**Comment by Akamaru** [comment]
> No problem for me for this anime but for "Wistoria Wand and Sword" I have this.

Yeah, looks like Nokotan fixed itself. It seems the bug is on the Crunhy side.

**Comment by Fu2chN** [comment]
> > No problem for me for this anime but for "Wistoria Wand and Sword" I have this.
> 
> Yeah, looks like Nokotan fixed itself. It seems the bug is on the Crunhy side.

An another app (CRD 1.4.32) download this anime with dubbing and subtitle. (Wistoria Wand and Sword)

**Comment by stratuma** [comment]
> > > No problem for me for this anime but for "Wistoria Wand and Sword" I have this.
> > 
> > 
> > Yeah, looks like Nokotan fixed itself. It seems the bug is on the Crunhy side.
> 
> An another app (CRD 1.4.32 aka "Crunchy-Downloader") download this anime with dubbing and subtitle. (Wistoria Wand and Sword)

Yeah because CRD only uses the english playlist. This bug only exists on non-english endpoint somehow.

**Comment by stratuma** [comment]
The only fix for this at the moment is to change the crunchyroll language in settings -> crunchy to english.

**Comment by TaMaGo-TTV** [comment]
Had the same issue, subs and dubs are now available for both mentioned Animes.

**Comment by Fu2chN** [comment]
I confirm, It's now work well.

**Comment by stratuma** [comment]
Yeah its always crunchyroll. They can't code.
